marquez/client.py

```diff
@@ -0,0 +1,260 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import os
+import urllib
+
+
+from marquez.constants import JobRunState
+from marquez.utils import request_utils
+from marquez.utils.request_utils import (InvalidRequestException,
+                                         handle_response)
+
+
+class MarquezClient(object):
+    API_PATH = "api/v1"
+    DEFAULT_TIMEOUT_SEC = 5
```

Comment:
I don't think we need both `DEFAULT_TIMEOUT_SEC` and `DEFAULT_TIMEOUT_MILLISECONDS`, with the latter should be enough.  Also the env variable could be named `MARQUEZ_TIMEOUT_MS` for more clarity.

Reply:
Fixed